presto-main/src/main/java/com/facebook/presto/sql/analyzer/SemanticErrorCode.java

```diff
@@ -49,6 +49,7 @@
     INVALID_LITERAL,
 
     FUNCTION_NOT_FOUND,
+    DUPLICATE_PARAMETER_DECLARATION,
```

Comment:
To be consistent with `DUPLICATE_COLUMN_NAME`, maybe name this to `DUPLICATE_PARAMETER_NAME`?